helm/values.yaml

```diff
@@ -61,17 +63,22 @@ hostAliases: {}
   #   region: us-east-2
 
 dnsPolicy: ""
-dnsConfig: {}
+dnsConfig:
+  {}
   # Example config which uses the AWS nameservers
   # dnsPolicy: "None"
   # dnsConfig:
   #   nameservers:
   #     - 169.254.169.253
 
 serviceAccount:
-  # Specifies whether a service account should be created
-  create: true
-  annotations: {}
-  ## Enable if EKS IAM for SA is used
-  #  eks.amazonaws.com/role-arn: arn:aws:iam::111122223333:role/efs-csi-role
-  name: efs-csi-controller-sa
+  controller:
```

Comment:
Hmm I'm thinking if this is considered a breaking change. Users might need to update their setup.

We need to update the chart version regardless, though.

Reply:
yes, we are going to go to 1.x.x